resqpy/rq_import/_GridFromCp.py

```diff
@@ -0,0 +1,555 @@
+"""_GridFromCp.py: Class to generate a RESQML grid object from an input corner point array, and other optional inputs."""
+
+version = '15th November 2021'
+
+# Nexus is a registered trademark of the Halliburton Company
+
+import logging
+
+log = logging.getLogger(__name__)
+log.debug('_GridFromCp.py version ' + version)
+
+import numpy as np
+import numpy.ma as ma
+
+import resqpy.crs as rqc
+import resqpy.grid as grr
+import resqpy.olio.vector_utilities as vec
+
+
+class GridFromCp:
+    """Class to build a resqpy grid from a Nexus CORP array"""
```

Comment:
This might still show up in the _readthedocs_ (maybe not). If it does, we should make it clear that the way to use this functionality is by calling the public grid_from_cp() function (for now), or a Grid._from_cp() class method (later).

Reply:
I've made this a private class now, so it shouldn't appear in the docs.